ha/const.py

```diff
@@ -194,8 +197,60 @@ class ACTION_STATUS(Enum):
     PENDING = "pending"
     COMPLETE = "complete"
 
-
 class INSTALLATION_TYPE(Enum):
     HW = "hw"
     VM = "vm"
     SINGLE_VM = "single_vm"
+
+# Alert attribute constants
+class ALERT_ATTRIBUTES:
```

Comment:
can we move this in event analyser
we can create event_analyser_terms.py
const is getting big and i think 
we should use module related const inside module and generic const here
which like mocroservice than monolithic

@indrajitzagade  @ajaysrivas want your opinion on this

Reply:
I like the approach and was thinking on similar lines. Let's start following it from next PRs. We have to do the refactor any way for existing constants. Another PR is pending due to this constant class.